2019-07-24-HW-FileReader/src/Demo1.java

```diff
@@ -27,8 +29,8 @@ public static void main(String[] args) {
 		
 	}
 	
-	public static void sumOfNumbers() { // #1
-		try (BufferedReader br1 = new BufferedReader(new FileReader("sumOfInt.txt"));) {
+	public static void sumOfNumbers(File file) { // #1
+		try (BufferedReader br1 = new BufferedReader(new FileReader(file));) {
 			System.out.println("1) Sum of numbers: " + sumOfInt(br1));
```

Comment:
your method sumIfNumbers should return a number. 
`public static int sumOfNumbers(File file)`
Do not leave println's in your code unless it is requried by the requirements. Otherwise it is just a debug info, which should not stay in the final code.